### PR TITLE
New Adapter: Pubrise

### PR DESCRIPTION
**Pull request by Pubrise**
doc - https://github.com/prebid/prebid.github.io/pull/5520

### CODE REVIEW

File: static/bidder-info/pubrise.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+endpoint: "https://backend.pubrise.ai/"
+maintainer:
+  email: "prebid@pubrise.ai"
```
Comment (gargcreation1992) [inline static/bidder-info/pubrise.yaml:3]: sent email for verification. Please reply with received.

File: static/bidder-info/pubrise.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    - banner
+    - video
+    - native
+  app:
+    mediaTypes:
+    - banner
+    - video
+    - native
+userSync:
+  redirect:
```
Comment (gargcreation1992) [inline static/bidder-info/pubrise.yaml:16]: User sync is not working. It's returning 404 for the following request. Please re-check.
```
https://sync.pubrise.ai/pbserverIframe?gdpr=&gdpr_consent=&ccpa=&gpp=&gpp_sid=&pbserverUrl=https%3A%2F%2Fib.adnxs.com%2Fprebid%2Fsetuid%3Fbidder%3Dpubrise%26gdpr%3D%26gdpr_consent%3D%26gpp%3D%26gpp_sid%3D%26f%3Db%26uid%3D%5BUID%5D
```

File: static/bidder-info/pubrise.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+endpoint: "https://backend.pubrise.ai/"
```
Comment (gargcreation1992) [inline static/bidder-info/pubrise.yaml:1]: endpoint is reachable.

**Comment by gargcreation1992** [comment]
@Pubrise Requesting you to please address the open comments to proceed further on this PR.

File: static/bidder-info/pubrise.yaml (hunk 2 above)
Comment (Pubrise) [reply static/bidder-info/pubrise.yaml:16]: fixed

**Comment by Pubrise** [comment]
@gargcreation1992 thanks, fixed user sync link, and confirmed email

File: static/bidder-info/pubrise.yaml (hunk 2 above)
Comment (gargcreation1992) [reply static/bidder-info/pubrise.yaml:16]: Working now. thanks!

**Review by gargcreation1992 — APPROVED** [review]

**Pull request merged**